ortho_seq_code/tests/test_orthoseqs.py

```diff
@@ -15,6 +15,7 @@ def test_cli(protein_seqs_no_padding, protein_pheno_no_padding):
     poly_order = "first"
```

Comment:
remove the above ~$ temporary file and add them to .gitignore as mentioned above